### PR TITLE
Fix branch name

### PR DESCRIPTION
**Pull request by KonstHardy**
Fix branch name (master was renamed to main):

master -> main

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -89,4 +89,4 @@
 
 ## Before 15.5.0
 
-PropTypes was previously included in React, but is now a separate package. For earlier history of PropTypes [see the React change log.](https://github.com/facebook/react/blob/master/CHANGELOG.md)
+PropTypes was previously included in React, but is now a separate package. For earlier history of PropTypes [see the React change log.](https://github.com/facebook/react/blob/main/CHANGELOG.md)
```
Comment (ljharb) [inline CHANGELOG.md:92]:
```suggestion
PropTypes was previously included in React, but is now a separate package. For earlier history of PropTypes [see the React change log.](https://github.com/facebook/react/blob/HEAD/CHANGELOG.md)
```
that way we don't have to hardcode the branch name

File: README.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ MyComponent.propTypes = {
 
   // A React element type (eg. MyComponent).
   // a function, string, or "element-like" object (eg. React.Fragment, Suspense, etc.)
-  // see https://github.com/facebook/react/blob/master/packages/shared/isValidElementType.js
+  // see https://github.com/facebook/react/blob/main/packages/shared/isValidElementType.js
```
Comment (ljharb) [inline README.md:87]:
```suggestion
  // see https://github.com/facebook/react/blob/HEAD/packages/shared/isValidElementType.js
```

**Review by ljharb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (KonstHardy) [reply CHANGELOG.md]: Yes, it's better this way. Thanks! :)

**Review by ljharb — APPROVED** [review]
thanks!

**Pull request merged**